index.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
       if (google.key) {
         params.push('key=' + encodeURIComponent(google.key));
       }
+      if (typeof google.libraries !== 'undefined' && google.libraries.length > 0) {
+        params.push('libraries=' + google.libraries.join());
```

Comment:
I have a branch where I have this too, to handle geolocation. Anyway, you need to join with `','` and also to escape it.